package.json

```diff
@@ -37,6 +38,7 @@
     "node": ">= 0.12"
   },
   "dependencies": {
+    "@types/node": "^12.0.8",
```

Comment:
@mapleeit So it's necessary "evil" to have this as prod dependency?

Reply:
Yes, I think it is...

Otherwise people need install this by themselves when using types.

Reply:
When they're using types, they should have it already from TS, right?

Reply:
I'll test this later.